src/chrome/content/rules/propub3r6espa33w.onion.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+	For related clearnet rules, see ProPublica.org.xml
+
+	Invalid certificate:
+		ppasset42kropoy6.onion (redirects to propub3r6espa33w.onion)
+	
+	Refused:
+		pubapp7v22ykdou3.onion (redirects to clearnet)
+
+-->
+<ruleset name="ProPublica (onion)">
+
+	<target host="propub3r6espa33w.onion" />
+	<target host="www.propub3r6espa33w.onion" />
+
```

Comment:
See also `pubdrop4dw6rk3aq.onion`.

Reply:
It's down at the moment. Was it working last time you checked it?

Reply:
@Bisaloo I did not check any `.onion` domains.